disputer/disputer.js

```diff
@@ -197,6 +197,13 @@ class Disputer {
       let withdrawAmount;
       try {
         withdrawAmount = await withdraw.call({ from: this.account });
+        // See: https://github.com/ethereum/solidity/issues/4840 for the reason this fix is necessary.
+        if (
+          withdrawAmount.rawValue.toString() ===
```

Comment:
Look at `common/ContractUtils` for a `revertWrapper` module you can use that checks for this hard-coded value. We use it in `reporters/GlobalSummaryReporter` to check if a price has resolved for example.

Reply:
ye, I agree would be ideal to not have this hardcoded here like this if it's something we deal with in other contexts.

Reply:
Yeah, the initial plan was to just get it merged as fast as possible to patch the live bots. But now that we've done the hot fix, I'll move this logic elsewhere.

Reply:
Done.